packages/mobx-undecorate/src/undecorate.ts

```diff
@@ -399,18 +399,19 @@ export default function transform(
     }
 
     function warn(msg: string, node: Node) {
-        if (process.env.NODE_ENV === "test") {
-            return
+        if (node.loc) {
+            const line = lines[node.loc!.start.line - 1]
+            const shortline = line.replace(/^\s*/, "")
+            console.warn(
+                `[mobx:undecorate] ${msg} at (${fileInfo.path}:${node.loc!.start.line}:${
+                    node.loc!.start.column
+                }):\n\t${shortline}\n\t${"^".padStart(
+                    node.loc!.start.column + 1 - line.indexOf(shortline),
+                    " "
```

Comment:
No need to include those when checking it above.
```suggestion
                    node.loc.start.column
                }):\n\t${shortline}\n\t${"^".padStart(
                    node.loc.start.column + 1 - line.indexOf(shortline),
                    " "
```

Reply:
Sorry, I forgot it. 😃